===================================================================
RCS file: /usr/local/lyx/cvsroot/lyx-devel/autogen.sh,v
retrieving revision 1.42
retrieving revision 1.42.2.1
diff -u -r1.42 -r1.42.2.1
--- autogen.sh	2002/12/23 14:14:12	1.42
+++ autogen.sh	2003/02/11 13:07:44	1.42.2.1
@@ -19,7 +19,7 @@
 	rm -f lib/reLyX/configure.ac
 	EXTRA_ACINCLUDE_FILES="lyxinclude213.m4"
 	;;
-    *2.5[2346])
+    *2.5[23467])
 	rm -f acconfig.h
 	rm -f configure.in
 	cp config/configure.ac .
